main.go

```diff
@@ -6,27 +6,113 @@ import (
 	"log"
 	"net/http"
 	"regexp"
+	"strings"
 )
 
 func main() {
-	// Generate a new word for hangman
+	willRepeat := true
+	for willRepeat {
+		var (
+			word         = generateWord()
+			obscuredWord = obscureWord(word)
+			userInput    = ""
+			lives        = 6
+		)
+
+		fmt.Println("Welcome to Hangman!")
+		fmt.Printf("You have %d lives to guess the word. \n", lives)
```

Comment:
nit: since we're using a `\n` here anyway, you might as well just use `fmt.Println()` again